src/ProdsStreamer.class.php

```diff
@@ -30,7 +30,7 @@ class ProdsStreamer
 	 */
 	private $file;
 
-	
+	public $metadata;
```

Comment:
perhaps @mboudet  can explain? i just based my PR on his branch

Reply:
If I recall correctly, I was looking for a way to pass the irods metadata to another service when using a stream. (So I couldn't directly get the metadata). I may have done this to pass the irods metadata through the php stream_get_meta_data() method.. Not 100% sure however..
This can be removed safely, though there are no way the get the irods metadata when using streams.